src/ts/ts-genapi/ci.yml

```diff
@@ -0,0 +1,100 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Can we put these files under tools and/or apiview? As it stands it isn't clear that this is used for ApiView.

Reply:
Also can probably improve the "ts-genapi" name a little to make it clearer what it is for.